src/content/docs/infrastructure/host-integrations/host-integrations-list/sendmail-monitoring-integration.mdx

```diff
@@ -18,188 +18,181 @@ Our Sendmail integration monitors the performance of your Sendmail mail transfer
+                split: horizontal
+                header_split_by: \s+(\d+)\s+(.*.)
+                set_header: [msgCount.latest, recipients.latest]
+                regex_match: true
+                split_by: \s+(\d+)\s+(.*.)
+
+            #Read the recipients host/domain.
+            - event_type: SendmailRecipientsHostname
+            commands:
+                - run: cat /var/log/mail.log | grep "to=<.*.>" | awk '{print $7}' | grep -oE '[^@]+[a-zA-Z0-9\-\.]+\.[a-zA-Z]{2,}' | sort | uniq -c
+                split: horizontal
+                header_split_by: \s+(\d+)\s+(.*.)
+                set_header: [hostCount.total, hostname.total]
+                regex_match: true
+                split_by: \s+(\d+)\s+(.*.)
+                - run: cat /var/log/mail.log | grep "to=<.*.>" | grep "$(date +'%b %e')" | awk '{print $7}' | grep -oE '[^@]+[a-zA-Z0-9\-\.]+\.[a-zA-Z]{2,}' | sort | uniq -c
+                split: horizontal
+                header_split_by: \s+(\d+)\s+(.*.)
+                set_header: [hostCount.latest, hostname.latest]
+                regex_match: true
+                split_by: \s+(\d+)\s+(.*.)
+
+            #Read the senders with message count.
+            - event_type: SendmailSendersbyMessageCount
+            commands:
+                - run: cat /var/log/mail.log | grep "from=<.*@.*>" | awk '{print $7}' | cut -d "<" -f2 |cut -d ">" -f1 | sort | uniq -c
+                split: horizontal
+                header_split_by: \s+(\d+)\s+(.*.)
+                set_header: [msg.total, senders.total]
+                regex_match: true
+                split_by: \s+(\d+)\s+(.*.)
+                - run: cat /var/log/mail.log | grep "from=<.*@.*>" | grep "$(date +'%b %e')" | awk '{print $7}' | cut -d "<" -f2 |cut -d ">" -f1 | sort | uniq -c
+                split: horizontal
+                header_split_by: \s+(\d+)\s+(.*.)
+                set_header: [msg.latest, senders.latest]
+                regex_match: true
+                split_by: \s+(\d+)\s+(.*.)
+
+            #Read the senders host/domain.
+            - event_type: SendmailSendersHostname
+            commands:
+                - run: cat /var/log/mail.log | grep "from=<.*.>" | awk '{print $7}' | grep -oE '[^@]+[a-zA-Z0-9\-\.]+\.[a-zA-Z]{2,}' | sort | uniq -c
+                split: horizontal
+                header_split_by: \s+(\d+)\s+(.*.)
+                set_header: [hostCount.total, hostname.total]
+                regex_match: true
+                split_by: \s+(\d+)\s+(.*.)
+                - run: cat /var/log/mail.log | grep "from=<.*.>" | grep "$(date +'%b %e')" | awk '{print $7}' | grep -oE '[^@]+[a-zA-Z0-9\-\.]+\.[a-zA-Z]{2,}' | sort | uniq -c
+                split: horizontal
+                header_split_by: \s+(\d+)\s+(.*.)
+                set_header: [hostCount.latest, hostname.latest]
+                regex_match: true
+                split_by: \s+(\d+)\s+(.*.) 
+    ```   
```

Comment:
in the preview, this is appearing in the codeblock. very small buggy behavior, could be related to indents?